pkgs/development/compilers/cudatoolkit/versions.toml

```diff
@@ -44,13 +44,13 @@ gcc = "gcc9"
 version = "11.4.2"
 url = "https://developer.download.nvidia.com/compute/cuda/11.4.2/local_installers/cuda_11.4.2_470.57.02_linux.run"
 sha256 = "sha256-u9h8oOkT+DdFSnljZ0c1E83e9VUILk2G7Zo4ZZzIHwo="
-gcc = "gcc11"
+gcc = "gcc10"
 
 ["11.5"]
 version = "11.5.0"
 url = "https://developer.download.nvidia.com/compute/cuda/11.5.0/local_installers/cuda_11.5.0_495.29.05_linux.run"
 sha256 = "sha256-rgoWk9lJfPPYHmlIlD43lGNpANtxyY1Y7v2sr38aHkw="
-gcc = "gcc11"
+gcc = "gcc10" # cuda 11.5 has problems with glibc 2.4 -> keeping gcc10
```

Comment:
Would you mind also adding a link to that NVIDIA forums discussion?

Reply:
There!